JetSubStructureMomentTools/Root/BosonTagTool.cxx

```diff
@@ -0,0 +1,24 @@
+#include "JetSubStructureMomentTools/BosonTagTool.h"
+#include "JetSubStructureUtils/BosonTag.h"
+
+using namespace std;
+
+BosonTagTool::BosonTagTool(std::string name) :
+  JetSubStructureMomentToolsBase(name)
+{
+  declareProperty("WorkingPoint", m_working_point               = "medium");
+  declareProperty("TaggerAlgorithm", m_tagger_alg               = "tau21");
+  declareProperty("RecommendationsFile", m_recommendations_file = "$ROOTCOREBIN/data/JetSubStructureUtils/config_13TeV_20150306.dat");
+  declareProperty("ApplyMassCut", m_compare_mass                = true);
+  declareProperty("ApplyVariableCut", m_compare_variable        = true);
+  declareProperty("EnableSimpleTagger", m_simple_tagger         = false);
+  declareProperty("Debug", m_debug                              = false);
+}
```

Comment:
Defaults here are the same as https://github.com/kratsg/JetSubstructureTools/blob/addWTagger/JetSubStructureUtils/JetSubStructureUtils/BosonTag.h#L59-L65